src/components/Integrations/EmptyState.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Button,
+  ButtonVariant,
+  Card,
+  CardBody,
+  CardHeader,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Gallery,
+  Icon,
+  Text,
+  TextContent,
+  TextVariants,
+  Title,
+  TitleSizes,
+} from '@patternfly/react-core';
+import {
+  ArrowRightIcon,
+  CogIcon,
+  CubesIcon,
+  HelpIcon,
+  InfrastructureIcon,
+} from '@patternfly/react-icons';
+import React from 'react';
+
+import { Messages } from '../../properties/Messages';
+import { Link } from 'react-router-dom';
+import { useAppContext } from '../../app/AppContext';
+
+const getEmptyIntegrationCard = (props) => {
+  const { TitleIcon, title, body, link } = props;
+  return (
+    <Card isFullHeight isPlain>
+      <CardHeader>
+        <Icon className="pf-v5-u-pl-sm pf-v5-u-pr-lg">
+          <TitleIcon className="pf-v5-u-link-color" />
+        </Icon>
+        <TextContent>
+          <Text component={TextVariants.h4}>{title}</Text>
+        </TextContent>
+      </CardHeader>
+      <CardBody className="pf-v5-u-display-flex">
+        <TextContent className="pf-v5-u-display-flex pf-v5-u-flex-direction-column pf-v5-u-align-self-stretch">
+          <Text component={TextVariants.p} className="pf-u-flex-grow-1">
+            {body}
+          </Text>
+          <Text component={TextVariants.p}>
+            <Link to={link} target="_blank">
+              Learn more
+              <Icon className="pf-u-ml-sm" isInline>
+                <ArrowRightIcon />
+              </Icon>
+            </Link>
+          </Text>
+        </TextContent>
+      </CardBody>
+    </Card>
+  );
+};
+
+export const IntegrationsEmptyState: React.FunctionComponent<{
+  onAddIntegration: (() => void) | undefined;
+}> = ({ onAddIntegration }) => {
+  const { isOrgAdmin } = useAppContext();
+  return (
+    <div className="pf-v5-l-flex pf-m-justify-content-center">
+      <EmptyState>
+        <EmptyStateIcon icon={CubesIcon} />
+        <Title headingLevel={TextVariants.h4} size={TitleSizes.lg}>
+          No integrations yet
+        </Title>
+        <EmptyStateBody>
+          <Gallery
+            aria-label="Card container"
+            minWidths={{ default: '100%', md: isOrgAdmin ? '30%' : '40%' }}
```

Comment:
This styling is not ideal - but works as expected. I've tried switching to flex, however that has it's own set of bugs (odd wrapping).